web/src/components/molecules/Asset/Asset/AssetBody/svgPreview.tsx

```diff
@@ -6,12 +6,15 @@ type Props = { url: string; svgRender: boolean };
 const SVGPreview: React.FC<Props> = ({ url, svgRender }) => {
   const [svgText, setSvgText] = useState("");
 
-  const fetchData = useCallback(() => {
-    fetch(url)
-      .then(res => res.text())
-      .then(val => {
-        setSvgText(val);
-      });
+  const fetchData = useCallback(async () => {
+    const res = await fetch(url, {
+      method: "GET",
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+      },
+    });
+    const text = await res.text();
+    setSvgText(text);
```

Comment:
Does this not through an error?  Doesn't fetch return an object with status and everything that you should check if the status is 200 and all that? Its been a while since I've used fetch...

All that being said, is it actually necessary to go through the trouble of fetching this JUST for fallback text? Why do you want to display the svg text and not just a "Could not display svg"?

Reply:
Do you mean "throw" an error? 
Do you mean that I should add an if statement to check the response.status?
however I added this line of code just in case:
    ```
    if (res.status !== 200) {
      setSvgText("Could not display svg");
      return;
    }
    ```

Reply:
That I think is better, thanks!